package.json

```diff
@@ -47,6 +47,14 @@
     "rollup": "^4.40.2",
     "standard": "^17.1.2"
   },
+  "pnpm": {
+    "overrides": {
+      "typescript": "^5.8.3"
+    },
```

Comment:
Not strictly needed but I'm annoyed that `api-extractor` is using its own bundled version of TypeScript 5.8.2